network/rpc/server/src/nova.rs

```diff
@@ -0,0 +1,84 @@
+use std::path::PathBuf;
+
+use nexus_config::{
+    vm::{NovaImpl, ProverImpl, VmConfig},
+    Config, MiscConfig,
+};
+use nexus_prover::types::{IVCProof, SeqPP};
+use nexus_rpc_common::ElfBytes;
+use nexus_vm::{init_vm, memory::trie::MerkleTrie, parse_elf_bytes, trace::trace};
+
+use super::{Error, ProverT};
+
+const NOVA_IMPL: NovaImpl = NovaImpl::Sequential;
+const CONFIG: VmConfig = VmConfig {
+    k: 16,
```

Comment:
Should this be configurable by the server as part of setup?

We could even imagine interfaces where there is universal/transparent setup where it makes sense to be part of the client request.

Reply:
Prover is configurable, you can replace it with jolt or even a dummy prover if needed.

Nova with some numbers is there for the purpose of running client example.